src/gdf_table.cuh

```diff
@@ -170,7 +183,10 @@ public:
         printf("Attempted to compare columns of different types.\n");
         return false;
       }
-
+      bool valid = gdf::util::get_bit(d_valids_data[i], my_row_index) && gdf::util::get_bit(other.d_valids_data[i], other_row_index);
```

Comment:
This logic will need to be updated.

If a column's value is NULL, then we need to apply the following logic
``` NULL == x iff (x == NULL)```

Therefore, the logic table would be:
- (0, 0) == continue to next column
- (0, 1) == return false
- (1, 0) == return false
- (1, 1) == continue with value comparison

This should be the only major change necessary to implement handling NULLs for the hash joins.